flow/actorcompiler/ActorCompiler.cs

```diff
@@ -86,31 +86,65 @@ class Function
         string indentation;
         StreamWriter body;
         public bool wasCalled { get; protected set; }
+        public Function overload = null;
```

Comment:
I'm not very familiar with this part (or most parts) of the actor compiler, but it's not very clear to me why this variable is called "overload".  Does it always represent the same sort of overload, meaning is it always either null or a function like the one its owner but with one parameter type changed to an rvalue reference?  Or is more of a general facility for having an overload for a function, so you can a chain of functions that are overloads of each other with each .overload pointing to the next?

Reply:
This `overload` field could in theory be used as you described, to create a linked list of multiple overloads. In practice, the only overload that ever gets generated is the rvalue reference overload, so `overload` is either `null` or a function like its owner but with one parameter type changed to an rvalue reference.